lib/LedgerSMB/Template/ODS.pm

```diff
@@ -147,11 +147,12 @@ sub _worksheet_handler {
     } else {
         $sheet = $ods->appendTable($_->{att}->{name}, $rows, $columns);
     }
+    return;
 }
 
 sub _row_handler {
     $rowcount++;
-    $currcol = 0;
+    return $currcol = 0;
```

Comment:
This callback and the ones below are probably expected to return undef: `return;`